lib/linters/decimal_zero.js

```diff
@@ -1,89 +1,89 @@
 'use strict';
 
-var isValidTrailing = function (number) {
-    var regexp = /^-?([^0]\d*)?\.(\d*0)$/;
-    return regexp.test(number);
-};
+module.exports = {
 
-var isValidLeading = function (number) {
-    var regexp = /^-?(0|\d+)\.(\d*[^0])$/;
-    return regexp.test(number);
-};
+    name: 'decimalZero',
+    nodeTypes: ['declaration'],
+    message: '%s should be written %s %s zero.',
 
-var isValidBoth = function (number) {
-    var regexp = /^-?(0|\d+)\.(\d*0)$/;
-    return regexp.test(number);
-};
+    lint: function decimalZeroLinter (config, node) {
 
-var isValidNone = function (number) {
-    var regexp = /^-?([^0]\d*)?\.(\d*[^0])$/;
-    return regexp.test(number);
-};
+        function isValidTrailing (number) {
```

Comment:
I think we should move all of these function definitions outside, bit cleaner IMO.